### PR TITLE
Update AppView.vue - Auto add app name to ENV for Minecraft

### PR DESCRIPTION
**Pull request by flikites**
Add APP_NAME ENV to minecraft marketplace deployments automatically.

### CODE REVIEW

**Review by Cabecinha84 — APPROVED** [review]
ack

File: HomeUI/src/views/apps/marketplace/AppView.vue [hunk 1]
```diff
@@ -802,6 +803,9 @@ export default {
           component.userEnvironmentParameters.forEach((param) => {
             envParams.push(`${param.name}=${param.value}`);
           });
+          if(props.appData.name.toLowerCase() === "minecraft") {
```
Comment (TheTrunk) [inline HomeUI/src/views/apps/marketplace/AppView.vue:806]: space eslint

**Review by TheTrunk — COMMENTED** [review]
just eslint error otherwise tack.

File: HomeUI/src/views/apps/marketplace/AppView.vue (hunk 1 above)
Comment (Cabecinha84) [reply HomeUI/src/views/apps/marketplace/AppView.vue:806]: already fixed on dev branch :)

**Pull request merged**